liquibase-integration-tests/src/test/groovy/liquibase/command/core/GenerateChangeLogMSSQLCommandTest.groovy

```diff
@@ -3,6 +3,7 @@ package liquibase.command.core
 import liquibase.Scope
 import liquibase.command.CommandResultsBuilder
 import liquibase.command.CommandScope
+import liquibase.command.core.helpers.DbUrlConnectionCommandStep
```

Comment:
It looks like the majority of tests in the GenerateChangeLog[DB]CommandTest.groovy classes are to validate specific regressions. I suggest adding a broader `generate-changelog` test that deploys a wider variety of objects to the database prior to executing `generate-changelog`. To be a bit more specific, I'm thinking of a test using a changelog more like this one: 

[liquibase](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba)/[liquibase-integration-tests](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests)/[src](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests/src)/[test](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests/src/test)/[resources](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests/src/test/resources)/[changelogs](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests/src/test/resources/changelogs)/[common](https://github.com/liquibase/liquibase/tree/19a15f51018eb21f0571b55d1ec820df59d0adba/liquibase-integration-tests/src/test/resources/changelogs/common)/common.tests.changelog.xml

QA also has changelogs that we can share, if that would be helpful.